src/index.js

```diff
@@ -3,7 +3,7 @@
 const mh = require('multihashes')
 const multibase = require('multibase')
 const multicodec = require('multicodec')
-const { baseTable: codecs } = require('multicodec/src/base-table.js')
+const { baseTable: codecs } = require('multicodec/src/generated-table')
```

Comment:
We should stop using non-public APIs. We broke the world once, I don't want it to break it again.

If there is no code to name mapping in js-multicodec, we should add it properly to the public API. I can have a look later this week.

Reply:
Cids exports code to name?

Reply:
AFAICT it only changed the file name nothing else